public/app/features/alerting/unified/api/alertmanager.ts

```diff
@@ -1,5 +1,12 @@
 import { getBackendSrv } from '@grafana/runtime';
-import { AlertManagerCortexConfig } from 'app/plugins/datasource/alertmanager/types';
+import {
+  AlertmanagerAlert,
+  AlertManagerCortexConfig,
+  AlertmanagerGroup,
+  Silence,
+  SilenceCreatePayload,
+  SilenceMatcher,
+} from 'app/plugins/datasource/alertmanager/types';
 import { getDatasourceAPIId, GRAFANA_RULES_SOURCE_NAME } from '../utils/datasource';
 
 // "grafana" for grafana-managed, otherwise a datasource name
```

Comment:
Are we missing the endpoint for an individual silence or is that not used in the UI?

Reply:
no use case in the frontend to fetch individual silences by ID at the moment

Reply:
Would be good to add it at some point, it's quite useful to look at an individual silence so that we can know which alerts are being affected by it - I know there's some of that on the main page but the other way around is also quite convenient.

Reply:
You will be able to see affected alerts of a silence in the list view:
https://www.figma.com/file/2s7EGevGeI4GuiArOmUECK/Unified-Alerting?node-id=273%3A721